### ISSUE TITLE
Build does not work on paths using symbolic links

### ISSUE THREAD

**Issue by francesquini** [noseglid/atom-build#55]
Simple test scenario:

```bash
/tmp$ mkdir test
/tmp$ ln -s test test_link
/tmp$ cd test
/tmp/test$ cat > .atom-build.json
{
  "cmd": "cat",
  "args": [ "{FILE_ACTIVE}" ],
  "sh": false
}
/tmp/test$ cat > whatever.abc
Hello World!
/tmp/test$
```
When atom is started from `/tmp/test`, and I try to build `whatever.abc` using "Ctrl+Alt+b" everything works as expected. However if atom is started from `/tmp/test_link` it doesn't do anything. In other words:

This works:
```bash
/tmp/test$ atom
```
This doesn't:
```bash
/tmp/test_link$ atom
```

My use case involves many shared files between projects which is done using symbolic links. Having a copy of everything for each project is something I'd like to avoid...

**Comment by noseglid** [comment]
Thanks for reporting this. I will look into this. Most likely `atom-build` must check the realpath of the working directory (i.e. resolve links) before testing for eligible build tools

**Comment by francesquini** [comment]
Thanks for the prompt response! Tell me if you need further details.

**Comment by h-no** [comment]
I have my project in a symbolic link and build doesn't start. It runs well if I open the project from it's actual location.
actual path: `/home/user/git/project`
sym link: `ln -s /home/user/git /git'`

**Comment by noseglid** [comment]
This has been fixed in the new version `0.28.0`. Thanks!

**Comment by h-no** [comment]
Works for me! Cheers
